core.js

```diff
@@ -547,6 +561,18 @@ async function _fromTokenizer(tokenizer) {
 		};
 	}
 
+	try {
```

Comment:
If the file signature is the same as PDF, why don't you move this into `if PDF ...`

Reply:
That... Hasn't crossed my mind in fact, and would probably solve the error that was thrown by the peekBuffer function! Will move in my next commit